### ISSUE TITLE
Solution for ng2 (Angular2)?

### ISSUE THREAD

**Issue by zarko-tg** [patrickmarabeas/ng-FitText.js#58]
Moving on to Angular 2...
Is there any good (somewhat proven) alternative to ng-FitText for the new framework at present?
Or a plan to develop this neat utility for it?

p.s. I see someone started https://github.com/lokenxo/ng2-fittext but it looks a bit too fresh out of the oven.

**Comment by patrickmarabeas** [comment]
I've moved away from Angular and haven't touched v2+ at all. Best case it'll just be some boilerplate alterations, at worst it'll require some learning on how things are best done now.

Happy to have a look when I get a free weekend soon. How hard can it be.

**Comment by trumbitta** [comment]
Hi @zarko-tg and @patrickmarabeas 

Since I've been in love with this directive for 2 years now, and I moved on to Angular 2+, I decided to port it:

https://github.com/ngTrumbitta/ngx-fit-text

It's in early stages, but it works in my test projects.

The difference with other similar projects out there, is that mine is a direct port of the technique and calculations used by ng-FitText.js, whereas the other ones are (mainly?) fresh approaches.

@patrickmarabeas let me know if you are happy with the acknowledgements or you'd like me to edit them somehow.
